tests/transforms/test_mitigate.py

```diff
@@ -115,6 +115,18 @@ def test_extrapolate_call(self, mocker):
         for t in tapes:
             same_tape(t, tape)
 
+    def test_shots_preserved(self):
+        """Tests that the mitigated circuits contain the same shots as the original circuit"""
+
+        w1, w2 = [np.random.random(2) for _ in range(2)]
+        _tape = qml.tape.QuantumScript(
+            [qml.SimplifiedTwoDesign(w1, w2, wires=range(2))],
+            [qml.expval(qml.PauliZ(0))],
+            shots=1000,
+        )
```

Comment:
A bit of an overkill as input example for a unit test 😄

Reply:
Haha true, harmless though?

Reply:
Updated it to a simpler example.

Reply:
true, the only harm would have been puzzlement of future readers of this test 😄